### PR TITLE
Update Bahn.de.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/Bahn.de.xml [hunk 1]
```diff
@@ -1,25 +1,19 @@
...
-  <target host=          "www.bahnshop.de" />
-
-  <rule from="^http://bahn\.de/"
-          to="https://www.bahn.de/" />
-  <rule from="^http:"
-          to="https:" />
+	<securecookie host=".+" name=".+" />
+	
+	<rule from="^http://bahn\.de/"
+		  to="https://www.bahn.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Bahn.de.xml]: ^bahn.de WFM

File: src/chrome/content/rules/Bahn.de.xml [hunk 2]
```diff
@@ -1,25 +1,19 @@
+<ruleset name="Bahn.de">
+	<target host="bahn.de" />
+	<target host="www.bahn.de" />
+	<target host="fahrkarten.bahn.de" />
+	<target host="grosskunden.bahn.de" />
+	<target host="mobile.bahn.de" />
+	<target host="ps.bahn.de" />
+	<target host="reiseauskunft.bahn.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/Bahn.de.xml]: See also:
inside.bahn.de
bauarbeiten.bahn.de
community.bahn.de
persoenlicherfahrplan.bahn.de
karriere.bahn.de
stredax.bahn.de
www.fundservice.bahn.de
gate.bahn.de
www.m.bahn.de
dbsystems.bahn.de

File: src/chrome/content/rules/Bahn.de.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/Bahn.de.xml]: Try with HTTPSE disabled.

File: src/chrome/content/rules/Bahn.de.xml (hunk 2 above)
Comment (koops76) [inline src/chrome/content/rules/Bahn.de.xml]: Later.

File: src/chrome/content/rules/Bahn.de.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/Bahn.de.xml]: I can confirm that the cert chain is incomplete. Can you add a hint about this?

Comment (koops76) [inline src/chrome/content/rules/Bahn.de.xml]: @J0WI Done.

File: src/chrome/content/rules/Bahn.de.xml (hunk 2 above)
Comment (J0WI) [inline src/chrome/content/rules/Bahn.de.xml]: Would you like to fill another PR?

Comment (koops76) [inline src/chrome/content/rules/Bahn.de.xml]: Yes.

Comment (J0WI) [inline src/chrome/content/rules/Bahn.de.xml]: Okay, please ping me if you have it.

**Pull request merged**